packages/query/src/block-processor.ts

```diff
@@ -128,29 +128,17 @@ export class BlockProcessor implements BlockProcessorInterface {
       { retry: () => true },
     );
 
-    if (currentHeight === -1n) {
-      // In the `for` loop below, we only update validator infos once we've
-      // reached the latest known epoch. This means that, if a user is syncing
-      // for the first time, they could experience a broken UI until the latest
-      // known epoch is reached, since they may have delegation tokens but no
-      // validator info to go with them. So we'll update validator infos at the
-      // beginning of sync as well, and force the rest of sync to wait until
-      // it's done.
-      await this.updateValidatorInfos(0n);
```

Comment:
nice, not sure why we were even doing this

Reply:
@jessepinho would have more idea on how critical this is

if this promise was voided it might also be fine? at least it may be possible to grab compactblocks before the limit kicks in

unfortunately the batch metadata request does not exist. it may also be possible to just ratelimit our requests locally.

Reply:
Don't think we should delete this as then we wouldn't have validator info until the latest epoch is synced. But a `void` sounds reasonable.

> they could experience a broken UI until the latest known epoch is reached

This sounds like we should make our UI in the staking page more robust so it doesn't throw if there is missing validator info.

Reply:
after examining, it seems like nothing is broken - they just don't render with icon etc. so i have no concerns voiding this